open_spiel/games/twixt/twixtboard.h

```diff
@@ -0,0 +1,247 @@
+
+// Copyright 2019 DeepMind Technologies Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef OPEN_SPIEL_GAMES_TWIXT_TWIXTBOARD_H_
+#define OPEN_SPIEL_GAMES_TWIXT_TWIXTBOARD_H_
+
+#include <map>
+#include <string>
+#include <vector>
+#include <utility>
+#include <set>
+
+#include "open_spiel/games/twixt/twixtcell.h"
+#include "open_spiel/spiel.h"
+
+namespace open_spiel {
+namespace twixt {
+
+const int kMinBoardSize = 5;
+const int kMaxBoardSize = 24;
+const int kDefaultBoardSize = 8;
+
+const bool kDefaultAnsiColorOutput = true;
+
+const double kMinDiscount = 0.0;
+const double kMaxDiscount = 1.0;
+const double kDefaultDiscount = kMaxDiscount;
+
+// 8 link descriptors store the properties of a link direction
+struct {
+  Move offsets;  // offset of the target peg, e.g. (2, -1) for ENE
+  std::vector<std::pair<Move, int>> blockingLinks;
+} typedef LinkDescriptor;
+
+// Tensor has 2 * 3 planes of size bordSize * (boardSize-2)
+// see ObservationTensor
+const int kNumPlanes = 6;
+
+enum Result { kOpen, kRedWin, kBlueWin, kDraw };
+
+enum Color { kRedColor, kBlueColor, kEmpty, kOffBoard };
+
+// blockerMap stores set of blocking links for each link
+static std::map<Link, std::set<Link>> blockerMap;
+
+inline std::set<Link> *getBlockers(Link link) { return &blockerMap[link]; }
+
+inline void pushBlocker(Link link, Link blockedLink) {
```

Comment:
Function names should not be camel case. This should be PushBlocker. Similarly for all the function across the source files.

Reply:
switched to CamelCase for all methods except simple getters accoring to the style guide